stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,74 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 50M
+  uploadTimeout: "10m"
+  storeDir: "/tmp/fluent-bit/s3"
+  storeDirLimitSize: 0
+  s3KeyFormat: /fluent-bit-logs/$TAG/%Y/%m/%d/%H/%M/%S
+  s3KeyFormatTagDelimiters:
+  staticFilePath: false
+  usePutObject: false
+  roleArn:
+  endpoint:
+  stsEndpoint:
+  cannedAcl:
+  compression:
+  contentType:
+  sendContentMd5: false
+  autoRetryRequests: true
+  logKey:
+  preserveDataOrdering: true
+  storageClass:
+  retryLimit: 1
+  externalId:
+  # extraOutputs: |
+
+opensearch:
+  enabled: true
+  match: "*"
+  host:
+  port: "443"
+  tls: "on"
+  path: ""
+  bufferSize: "5m"
+  pipeline:
+  awsRegion: "us-east-1"
+  awsAuth: "On"
+  awsStsEndpoint:
+  awsRoleArn:
+  awsExternalId:
+  httpUser:
+  httpPasswd:
+  index: "aws-fluent-bit"
+  type: "_doc"
```

Comment:
Thank you for suppressing type name by default: https://docs.fluentbit.io/manual/pipeline/outputs/opensearch#action-metadata-contains-an-unknown-parameter-type

Since we do that, why set this value?

Reply:
Its just there as a default from fluent bit